target/linux/ramips/image/mt7621.mk

```diff
@@ -71,16 +72,18 @@ define Build/ubnt-erx-factory-image
 	fi
 endef
 
+# MTC WR1201
 # The OEM webinterface expects an kernel with initramfs which has the uImage
 # header field ih_name.
 # We don't wan't to set the header name field for the kernel include in the
 # sysupgrade image as well, as this image shouldn't be accepted by the OEM
 # webinterface. It will soft-brick the board.
-define Build/wr1201-factory-header
+define Build/custom-initramfs-uimage
```

Comment:
Maybe we can merge this with Build/uImage in image-commands.mk...?

Reply:
I want to set a special name only in initramfs image on WN-DX1167R and WN-AX1167GR2. On WR1201, probably the same reason. Currently, ```Build/uImage``` set the name to all images(initramfs, sysupgrade, factory, ...) by ```UIMAGE_NAME```.

Reply:
Oh, I didn't read the comment up there, sorry. Then no problem here.

Reply:
This node should be rearranged according to the new name.